cms/models_test.py

```diff
@@ -1819,3 +1819,42 @@ def test_certificate_request_with_invalid_uuid(user_client, uuid_string):
 
     course_certificate_resp = user_client.get(f"/certificate/{uuid_string}/")
     assert course_certificate_resp.status_code == 404
+
+
+def test_get_child_page_of_type_with_draft():

```

Comment:
We should pobably divide it into two tests:

1. The method that returns the draft pages
2. Another to validate that we can't add another draft page as child if one already exists? (We might not even need that since that's the functionality of Wagtail itself.)

Reply:
What are the things that you want me to split? There is a single assert to check that we cannot create a new page if one exists.

Reply:
All of the things are related to a single test.

Reply:
Oh, It was my perception that checking either the `child_pages_including_draft` returns draft child pages or not and checking if we can create another child page if a draft page already exists are two different things.

Reply:
Then we should probably give a suitable name to the test.